src/services/vision/client.ts

```diff
@@ -189,8 +189,17 @@ export class VisionClient implements Vision {
       pb.CaptureAllFromCameraRequest,
       pb.CaptureAllFromCameraResponse
     >(service.captureAllFromCamera.bind(service), request);
+
+    const image = response.getImage();
+
     return {
-      image: response.getImage()?.toObject(),
+      image: image
+        ? {
+            format: image.getFormat(),
+            sourceName: image.getSourceName(),
+            image: image.getImage_asU8(),
```

Comment:
Is the return value of this function strictly typed? If so, all good. If not, this image response could drift if someone changed the image definition (e.g. added a new field to `Image`)

Reply:
It is, it's following the definition of `Image` and a return type error will be thrown if it doesn't adhere to it.